locales/en-US/messages.properties

```diff
@@ -91,3 +91,52 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
 
 # Site title when shared on Facebook
 support_mozilla=Support Mozilla
+
+# faq page
+faq_header=FAQ
+faq_donate_link=CLICK HERE TO DONATE NOW
+faq_intro=Below are the most frequently asked questions about donating to the Mozilla Foundation. If your question and answer are not listed here, please contact us at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>.
+faq_item_1_header=What are the ways I can donate?
+faq_item_1_paragraph=<a href="https://donate.mozilla.org/">Visit this link to make a secure online donation right now via credit card or PayPal</a>. For additional ways to give -- like check, bitcoin, or bank transfer -- visit our <a href="https://wiki.mozilla.org/Ways_to_Give"Ways to Give page</a>.
```

Comment:
link tag not closed `<a href="https://wiki.mozilla.org/Ways_to_Give" _>_ Ways to Give page</a>`